Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -1362,3 +1378,43 @@ public struct DelayedImmutable<Value> {
         }
     }
 }
+
+extension Array where Element == PackageConditionProtocol {
+    func toPlatformFilters() -> [PIF.PlatformFilter] {
+        var result: [PIF.PlatformFilter] = []
+        let platformConditions = self.compactMap{ $0 as? PlatformsCondition }.flatMap{ $0.platforms }
+
+        for condition in platformConditions {
+            switch condition {
+            case .macOS:
+                result.append(.init(platform: "macos"))
+
+            case .iOS:
+                result.append(.init(platform: "ios"))
+                result.append(.init(platform: "ios", environment: "simulator"))
+
+            case .tvOS:
+                result.append(.init(platform: "tvos"))
+                result.append(.init(platform: "tvos", environment: "simulator"))
+
+            case .watchOS:
+                result.append(.init(platform: "watchos"))
+                result.append(.init(platform: "watchos", environment: "simulator"))
+
+            case .linux:
+                result.append(.init(platform: "linux"))
```

Comment:
Should also include a bunch of environments of common Linux ABIs:

    result.append(.init(platform: "linux"))
    result.append(.init(platform: "linux", environment: "eabi"))
    result.append(.init(platform: "linux", environment: "gnu"))
    result.append(.init(platform: "linux", environment: "gnueabi"))
    result.append(.init(platform: "linux", environment: "gnueabihf"))

We also shouldn't try to wildcard it, because we _don't_ want Linux to match Android, for example.